spa/variables.tf

```
@@ -71,6 +71,12 @@ variable "basic_auth_credentials" {
   default     = null
 }
 
+variable "basic_auth_exclusions" {
+  description = "List of regular expressions describing paths excluded from the basic auth"
+  type        = list(string)
+  default     = null
```

```suggestion
  default     = []
```

that way you won't have to add `null` checks

---

Good idea!